docs/api/networksocket/SocketStats.md

```diff
@@ -0,0 +1,21 @@
+## SocketStats
+
+Use the SocketStats class to read the network socket statistics. SocketStats Class object is created by "InternetSocket" class when statistics are enabled by `nsapi.socket-stats-enable` config option.
+
+The `SocketStats` class provides the static function `mbed_stats_socket_get_each` to get the network socket information for all sockets both currently in use and closed. Using this API you can know how many sockets are open, destination IP addresses, the amount of data sent and received in bytes, the type of socket and the timestamp of recent "socket state" changes recorded in `last_change_tick`.
```

Comment:
Please change "Using this API.." to "Using this function...". API usually refers to a set of functions/routines(courtesy: Kevin G). Everything else looks good to me.

Reply:
Done. 👍